closed/OpenJ9.gmk

```diff
@@ -182,6 +183,7 @@ $(foreach file, \
 		testlibA \
 		testlibB \
 		vmruntimestateagent29 \
+		clinkerffitests \
```

Comment:
Agreed and removed the entry in which case https://github.com/ibmruntimes/openj9-openjdk-jdk16/pull/53 must be merged at first to avoid breaking the tests.